main/models.py

```diff
@@ -119,14 +126,14 @@ def cashflows(self):
         for c in date_sorted:
             balance += c.amount
             result.append(
-                CashflowAndBalance(c.date, c.amount, c.description, balance))
+                CashflowAndBalance(c.date, round(c.amount,2), c.description, round(balance,2)))
```

Comment:
The rounding should only be made in 1 place (where you perform the partial rent computation)